common/djangoapps/student/views/login.py

```diff
@@ -386,6 +386,28 @@ def send_reactivation_email_for_user(user):
     return JsonResponse({"success": True})
 
 
+@ensure_csrf_cookie
+def verify_user_password(request):
+    """
+    If the user is logged in and we want to verify that they have submitted the correct password
+    for a major account change (for example, deleting this user account).
+
+    Args:
+        request (HttpRequest): A request object where the password should be included in the POST fields.
+    """
+    # TODO: Use the rate limiting code from above.
+    try:
+        log.info('username: %r', request.user.username)
```

Comment:
Is this log statement needed?  If so, you'll want to add further information in the log statement to provide Splunk viewers with additional context.